src/validations/productValidation.ts

```diff
@@ -1,5 +1,20 @@
 import { body, param, query } from "express-validator";
 
+export const validateGetAllProducts = [
+  query("limit")
+    .optional()
+    .isInt({ min: 1, max: 100 })
```

Comment:
Try to use some constant file for all hard coded numbers in the project! Import that file and use its constants where needed

Reply:
Yes, we opened a JIRA for that will be fixed in the upcoming sprint. Thank you.